Part 4 - Platform Features/README.md

```diff
@@ -157,7 +156,7 @@ This project is pre-configured with all required permissions and features needed
     {
         try
         {
-            await map.OpenAsync(Monkey.Latitude, Monkey.Longitude, new MapLaunchOptions
```

Comment:
This api was changed back. Make sure you are compiling against the latest GA build